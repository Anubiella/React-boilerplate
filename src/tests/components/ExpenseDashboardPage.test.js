import React from 'react';
import {shallow} from 'enzyme';
import DashboardPage from '../../components/DashboardPage';

test('should display the filters list and the expense list', () => {
    const wrapper = shallow(<DashboardPage />);
    expect(wrapper).toMatchSnapshot();
});