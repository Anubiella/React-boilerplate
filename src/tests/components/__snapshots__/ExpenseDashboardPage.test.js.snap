// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should display the filters list and the expense list 1`] = `
<div>
  Dashboardpage content
</div>
`;
